jekyll/_cci2/contexts.md

```diff
@@ -62,8 +62,9 @@ If you find you need to rename an org or repo that you have previously hooked up
       my-workflow:
         jobs:
           - run-tests:
-              context: org-global        
```

Comment:
presumably this way is still valid if using a single context? Also - can server customers use the syntax you've added here? Just wondering if we need separate examples...

Reply:
The old way is still valid, though I think it might be less confusing if there's only one set of semantics presented.

As discussed on Slack, server customers will have to wait until a handful of services have been updated in server, I assume that's going to happen with 3.0 or 3.1.

Reply:
👍 Yep I get that Server customers can't use the list of contexts, but can they add a single context as you show here?

Reply:
They cannot, at least not until at least orb-service has been updated in server :/